README.md

```diff
@@ -1,17 +1,77 @@
-1. Project info
-This project is about creating simple Tic Tac Toe game.
+
+TicTacToeGit
+
+Simple game of Tic Tac Toe 
+
+
+Getting Started
+
+/*These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.*/
+
 It suppose to:
 - enable  two players to have a game
 - display information whose turn is is
 - mark tiles with "X" or "0"
 - acknowledge end of the game by getting one of the winning combinations
 - display the result/winner
 
-2. Technology
+You can find a copy of the project here: https://github.com/Kesse8401/TicTacToe
+
+
+Prerequisites
+
+/* What things you need to install the software and how to install them */
```

Comment:
good call, you can list stuff like
* node (and the version you currently have)
* npm and its version